### ISSUE TITLE
Accessibility issue on resize button

### ISSUE THREAD

**Issue by antsteyeragorize** [hugerte/hugerte#99]
## 📝 Provide detailed reproduction steps (if any)

1. Go to https://hugerte.org/
2. Open dev console
3. Run Lighthouse with "Accessibility" checked

### ✔️ Expected result

HugeRTE should have a 100% lighthouse score

### ❌ Actual result

![Image](https://github.com/user-attachments/assets/92f61d7f-9837-4fdf-8329-e90c1366e3fd)

### ❓ Possible solution

Seems like the resize handle should have the role="button" in order to aria-label to work properly.

## 📃 Other details
- Browser: Chrome, Chromium
- OS: MacOS
- First affected version: …
- Worked in version: …

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by SentryAryan** [comment]
I am using this rte for my personal project , can you suggest a workaround?

**Comment by carlosmintfan** [comment]
@SentryAryan What do you mean by personal project? Is it private or public? Just because, if no disabled persons are using your project, you can ignore this, I think. I'll look into fixing this soon but am quite busy right now, so please give me some days.

**Comment by antsteyer** [comment]
Hi @SentryAryan the only way I found is to hide the button of resize, so it is not readable by screen readers.

```css
.tox-statusbar__resize-handle {
  display: none;
}
```

**Comment by antsteyer** [comment]
For reference: https://github.com/w3c/aria/issues/1348
